spec/index.html

```diff
@@ -1089,6 +1089,40 @@ <h2>Literal value</h2>
         same literal <a>RDF term</a> because their
         <a>lexical forms</a> differ.</p>
     </section>
+
+    <section id="section-text-direction" class="informative">
+      <h4>Initial text direction</h4>
```

Comment:
When used as a title, I believe common usage requires it be. “Initial Text Directiom”, even if it is “Initial text direction” in the body.